ci-operator/config/stackrox/scanner/stackrox-scanner-master.yaml

```diff
@@ -4,80 +4,17 @@ base_images:
     namespace: ocp
     tag: "8"
 build_root:
-  project_image:
-    dockerfile_path: .openshift-ci/Dockerfile.build_root
-images:
-- dockerfile_path: .openshift-ci/build/Dockerfile.generate-genesis-dump
-  from: src
-  to: genesis-dump
-- dockerfile_path: .openshift-ci/build/Dockerfile.generate-db-dump
-  from: genesis-dump
-  to: db-dump
-- dockerfile_path: .openshift-ci/build/Dockerfile.build-bundle
-  from: genesis-dump
-  to: scanner-bundle
-- dockerfile_path: .openshift-ci/build/Dockerfile.build-db-bundle
-  from: db-dump
-  to: scanner-db-bundle
-- dockerfile_path: Dockerfile
-  from: ubi-minimal
-  inputs:
-    scanner-bundle:
-      paths:
-      - destination_dir: .
-        source_path: /go/src/github.com/stackrox/scanner/image/scanner/rhel/.
-  to: scanner
-- dockerfile_path: Dockerfile.slim
-  from: ubi-minimal
-  inputs:
-    scanner-bundle:
-      paths:
-      - destination_dir: .
-        source_path: /go/src/github.com/stackrox/scanner/image/scanner/rhel/.
-  to: scanner-slim
-- dockerfile_path: Dockerfile
-  from: ubi-minimal
-  inputs:
-    scanner-db-bundle:
-      paths:
-      - destination_dir: .
-        source_path: /go/src/github.com/stackrox/scanner/image/db/rhel/.
-  to: scanner-db
-- dockerfile_path: Dockerfile.slim
-  from: ubi-minimal
-  inputs:
-    scanner-db-bundle:
-      paths:
-      - destination_dir: .
-        source_path: /go/src/github.com/stackrox/scanner/image/db/rhel/.
-  to: scanner-db-slim
+  image_stream_tag:
+    name: apollo-ci
+    namespace: stackrox
+    tag: stackrox-test-0.3.61
```

Comment:
We don't mirror scanner-test - should we?

Reply:
> We don't mirror scanner-test - should we?

I'd assume it is necessary as that is what the current environment uses via Dockerfile.build_root. 
That has to be a separate PR i.e. it needs to be mirrored before it can be used here.